### PR TITLE
added yarn stylelint and removed stylelint related errors

### PR DESCRIPTION
**Pull request by shoebsd31**
Signed-off-by: shoeb <shoebsd31@gmail.com>

## Resolves #186 

I have added yarn packages **stylelint** and **stylelint-config-standard**. I have also added a hidden .stylelintrc.json file to configure some setup to avoid usage of !important

### How to test

run the following command to test the changes
**yarn stylelint "**/*.{scss,vue}"**

### Screenshots

_Screenshots of the changes working_

### CODE REVIEW

**Comment by shoebsd31** [comment]
Please ignore my previous pull request. I had some issues with merging so I created a fresh new fork and updated the repository

**Comment by ann-kilzer** [comment]
Thanks for making these changes. At a quick glance, they look good, but I'll let @ermish take a look.

It might be useful to also call the `stylelint` command inside the `yarn lint` command, would you be able to add that?

File: components/CancellationList.vue [hunk 1]
```diff
@@ -246,20 +246,23 @@ export default {
...
+
 #note {
   padding-bottom: 20px;
 }
+
 #waiting-lists-table {
   margin-bottom: 20px;
 }
-
+/* stylelint-disable */
```
Comment (ermish) [inline components/CancellationList.vue:265]: I guessing it doesn't understand the `v-deep` here? Is there a stylelint rules plugin for vue to support this?

**Review by ermish — APPROVED** [review]
LGTM. \
I added a comment, but we fix in a separate PR.
Thanks for wrapping this up, @shoebsd31!!! 👾 💯

**Comment by shoebsd31** [comment]
> LGTM. \
> I added a comment, but we fix in a separate PR.
> Thanks for wrapping this up, @shoebsd31!!! 👾 💯 

Thanks @ermish

**Comment by ermish** [comment]
side note, cool portfolio @shoebsd31 😎

**Comment by ermish** [comment]
checked preview, site looks consistent 👍 merging

**Pull request merged**